### DISCUSSION TITLE
@0no-co/graphqlsp no longer needed when using TS 5.5+?

### DISCUSSION THREAD

**Discussion by o-az** [0no-co/GraphQLSP#340] [Q&A] [answered]
just curious if `@0no-co/graphqlsp` is no longer needed if I'm using TS 5.5+

**Comment by JoviDeCroock** [comment, marked as answer]
You can replace it with the `gql.tada/ts-plugin` one instead yes!

**Reply by Sleepful** [reply]
Oh, thanks for this.
For other newcomers: 
install `npm install gql.tada -D`,
add to tsconfig:
```json
    "plugins": [ 
      { 
        "name": "gql.tada/ts-plugin", 
        "schema": "./db/schema.graphql", 
        "tadaOutputLocation": "./src/graphql-env.d.ts"
      } 
    ] 
```
now run your typescript lsp in your editor (tsserver) and you shall find `./src/graphql-env.d.ts`, that means it worked
